features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/Certificates.jsx

```diff
@@ -235,9 +235,8 @@ function Certificates(props) {
                                         </IconButton>
                                         <IconButton
                                             onClick={() => setCertificateToDelete({ open: true, alias: cert.alias })}
-                                            color='secondary'
                                         >
-                                            <Icon>delete</Icon>
+                                            <Icon color='error'>delete</Icon>
```

Comment:
Can't we use the color as red instead?

Reply:
IMHO we should not use a different color for delete icon, what we need to have is either a conformation box or a way to revert the action for some pre defined time.